abstract/group.go

```diff
@@ -51,6 +51,9 @@ type Secret interface {
 
 	// Set to a fresh random or pseudo-random secret
 	Pick(rand cipher.Stream) Secret
+
+	// SetBytes will take bytes and create a secret out of it
+	SetBytes([]byte) Secret
```

Comment:
Isn't `MarshalBinary` compressing the integer if it's value allows it? I thought that:
- (Un)MarshalBinary are used to store things in a file - variable length, might have indexes and other stuff
- (Set)Bytes work directly on the value of the secret and always take/return the same length